cluster/addons/e2e-rbac-bindings/e2e-user-binding.yaml

```diff
@@ -1,19 +0,0 @@
-# This is the main user for the e2e tests.  This is ok to leave long term
-# since the first user in the test can reasonably be high power
-# its kubecfg in gce
-# TODO consider provisioning each test its namespace and giving it an
-# admin user.  This still has to exist, but e2e wouldn't normally use it
-apiVersion: rbac.authorization.k8s.io/v1alpha1
-kind: ClusterRoleBinding
-metadata:
-  name: e2e-user-cluster-admin
-  labels:
-    kubernetes.io/cluster-service: "true"
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: cluster-admin
-subjects:
-- apiVersion: rbac/v1alpha1
-  kind: User
-  name: kubecfg
```

Comment:
This is not needed in GKE. But this will probably break client cert access to GCE clusters that upgrade to 1.6.